.github/workflows/validate-licenses.yaml

```diff
@@ -14,7 +14,7 @@ on:
       - main
 jobs:
   check-license-yaml:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted-nutanix-docker-large
```

Comment:
Changing a single workflow to use different kind of runner will result in incompatible cache generated by devbox GHA for the existing ubuntu runners. This will require also turning off the cache or running settings the `refresh-cli` to `true` for the devbox GHA.

Example https://github.com/mesosphere/kommander-applications/actions/runs/18095323069/job/51484923099?pr=4066